lib/test.js

```diff
@@ -0,0 +1,13 @@
+const globby = require('globby');
```

Comment:
This repo is based on https://github.com/creately/ts-starter so we can use TypeScript inside `./src` directory. Let's move this file to `src` and use TypeScript.

Reply:
Done